src/languages/shortcode-completion.ts

```diff
@@ -33,16 +33,19 @@ export class ShortcodeCompletionProvider implements vscode.CompletionItemProvide
   }
 
   private isBeginningOfShortcode(linePrefix: string): boolean {
-    return linePrefix.endsWith('{{< ') || linePrefix.endsWith('{{% ');
+    return linePrefix.endsWith('{{< ') || linePrefix.endsWith('{{% ') || linePrefix.endsWith('{{<') || linePrefix.endsWith('{{%');
```

Comment:
This shouldn't work.

We will need to add `<` and `%` to the `triggerCharacter`, at least:
https://github.com/kofuk/vscode-hugo-utils/blob/4c3383d66d2087971acf5d36f5b034298a44ed51/src/extension.ts#L21